plugwise/smile.py

```diff
@@ -87,17 +131,25 @@ def get_all_devices(self) -> None:
             self._opentherm_device = open_therm_boiler is not None
 
             # Determine if the Adam or Anna has cooling capability
-            locator = "./gateway/features/cooling"
-            anna_cooling_present_1 = adam_cooling_present = (
-                self._domain_objects.find(locator) is not None
-            )
-            # Alternative method for the Anna with Elga
+            locator_1 = "./gateway/features/cooling"
             locator_2 = "./gateway/features/elga_support"
-            anna_cooling_present_2 = self._domain_objects.find(locator_2) is not None
-            if self.smile_name == "Anna":
-                self._anna_cooling_present = (
-                    anna_cooling_present_1 or anna_cooling_present_2
-                )
+            locator_3 = "./module[vendor_name='Atlantic']"
+            locator_4 = "./module[vendor_model='Loria']"
+            search = self._domain_objects
+            self._anna_cooling_present = adam_cooling_present = False
+            if search.find(locator_1) is not None:
+                if self.smile_name == "Anna":
+                    self._anna_cooling_present = True
+                else:
+                    adam_cooling_present = True
+            # Alternative method for the Anna with Elga
+            elif search.find(locator_2) is not None:
+                self._anna_cooling_present = True
+            # Alternative method for the Anna with Loria/Thermastage
+            elif search.find(locator_3) is not None:
```

Comment:
Shouldn't we do this in one line?

Reply:
Yes, but then then readability of the code becomes worse.
I'll apply the change so you can judge for yourself.

Reply:
Hmm, it's still readable, so let's leave it.